# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/KnowsCompositor.h
# Commit: 9d637c05af7f
# Full Hash: 9d637c05af7f8c39af78b290335ca55994849ee8
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-06 04:14:34
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add RenderTextureHostSWGL support to D3D11 RenderTextureHosts. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95799
# ==============================================================================

diff -r ea88b243717a -r 9d637c05af7f gfx/layers/ipc/KnowsCompositor.h
--- a/gfx/layers/ipc/KnowsCompositor.h	Fri Nov 06 00:00:47 2020 +0000
+++ b/gfx/layers/ipc/KnowsCompositor.h	Fri Nov 06 00:01:36 2020 +0000
@@ -103,7 +103,9 @@
   bool SupportsD3D11() const {
     return GetCompositorBackendType() == layers::LayersBackend::LAYERS_D3D11 ||
            (GetCompositorBackendType() == layers::LayersBackend::LAYERS_WR &&
-            GetCompositorUseANGLE());
+            (GetCompositorUseANGLE() ||
+             mTextureFactoryIdentifier.mWebRenderCompositor ==
+                 layers::WebRenderCompositor::D3D11));
   }
 
   bool GetCompositorUseANGLE() const {